### PR TITLE
REFS#132 and FIXES#132

### PR DESCRIPTION
**Pull request by viralgujarati**

### CODE REVIEW

File: Modules/SQL/Day6/Viral Gujarati/Assignment/Day6.sql [hunk 1]
```diff
@@ -0,0 +1,524 @@
...
+SELECT FirstName, LastName, salary
+FROM Employees 
+WHERE Salary > (SELECT AVG(Salary) FROM Employees);
+
+
+
+/*Find the names (first_name, last_name), salary of the employees whose salary is equal to the minimum salary for their job grade. */
+SELECT FirstName, LastName, Salary
+FROM Employees
+WHERE Salary IN (
```
Comment (radixdt12020) [inline Modules/SQL/Day6/Viral Gujarati/Assignment/Day6.sql:315]: why in operator is used?

File: Modules/SQL/Day6/Viral Gujarati/Assignment/Day6.sql [hunk 2]
```diff
@@ -0,0 +1,524 @@
...
+
+/*Find the employee ID, job title, number of days between ending date and starting date for all jobs in department 90 from job history. */
+SELECT EmployeeID, JobID, DATEDIFF(DAY, StartDate, EndDate) AS 'NoOfDays'
+FROM JobHistory
+WHERE DepartmentID=90
+
+/*Write a query to display the department ID, department name and manager first name. */
+SELECT d.DepartmentID, d.DepartmentName, e.FirstName AS 'ManagerName'
+FROM Departments AS d JOIN Employees AS e
+ON d.ManagerID = e.EmployeeID
```
Comment (radixdt12020) [inline Modules/SQL/Day6/Viral Gujarati/Assignment/Day6.sql:497]: d.employeeid =e.managerid

**Review by radixdt12020 — CHANGES_REQUESTED** [review]
do necessary modification

File: Modules/SQL/Day6/Viral Gujarati/Assignment/Day6.sql (hunk 2 above)
Comment (viralgujarati) [reply Modules/SQL/Day6/Viral Gujarati/Assignment/Day6.sql:497]: Yes

**Pull request merged**